afs-security-analysis-local/src/test/java/com/powsybl/afs/security/local/SecurityAnalysisProviderMock.java

```diff
@@ -0,0 +1,43 @@
+package com.powsybl.afs.security.local;
+
+import com.google.auto.service.AutoService;
+import com.powsybl.computation.ComputationManager;
+import com.powsybl.contingency.ContingenciesProvider;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.security.*;
+import com.powsybl.security.interceptors.SecurityAnalysisInterceptor;
+import com.powsybl.security.monitor.StateMonitor;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+@AutoService(SecurityAnalysisProvider.class)
```

Comment:
See comment in the test class: I suggest to not register it as a service, and simply provide it in the service constructor (this way we don't "pollute" the service loader).